examples/layers/rendering/src/sector_layout.dart

```diff
@@ -585,3 +586,48 @@ class RenderSolidColor extends RenderDecoratedSector {
     }
   }
 }
+
+/// The result of performing a hit test on [RenderSector]s.
+class SectorHitTestResult extends HitTestResult {
+  /// Creates an empty hit test result for hit testing on [RenderSector].
+  SectorHitTestResult() : super();
+
+  /// Wraps `result` to create a [HitTestResult] that implements the
+  /// [SectorHitTestResult] protocol for hit testing on [RenderSector]s.
+  ///
+  /// This method is used by [RenderObject]s that adapt between the
+  /// [RenderSector]-world and the non-[RenderSector]-world to convert a (subtype of)
+  /// [HitTestResult] to a [SectorHitTestResult] for hit testing on [RenderSector]s.
+  ///
+  /// The [HitTestEntry]s added to the returned [SectorHitTestResult] are also
+  /// added to the wrapped `result` (both share the same underlying data
+  /// structure to store [HitTestEntry]s).
+  ///
+  /// See also:
+  ///
+  ///  * [HitTestResult.wrap], which turns a [SectorHitTestResult] back into a
+  ///    generic [HitTestResult].
+  SectorHitTestResult.wrap(HitTestResult result) : super.wrap(result);
+}
+
+/// A hit test entry used by [RenderSector].
+class SectorHitTestEntry extends HitTestEntry {
+  /// Creates a box hit test entry.
+  ///
+  /// The [radius] and [theta] argument must not be null.
+  SectorHitTestEntry(RenderSector target, { @required this.radius,  @required this.theta })
+      : assert(radius != null),
+        assert(theta != null),
+        super(target);
+
+  @override
+  RenderSector get target => super.target;
```

Comment:
That probably would have been a good idea when HitTestEntry was first written. This is now just copying what the existing BoxHitTestEntry and SliverHitTestEntry is doing. Changing this now would be another breaking change. This one I find really hard to justify since it doesn't provide any user value. I'll leave it as is for now.